docs_user/modules/proc_preparing-block-storage-service-by-customizing-configuration.adoc

```diff
@@ -13,16 +13,16 @@ The high level explanation of the tailor-made approach is:
 endif::[]
 
 .Procedure
-. Determine what part of the configuration is generic for all the Block Storage services and remove anything that would change when deployed in {rhocp_long}, such as the `connection` in the `[database]` section, the `transport_url` and `log_dir` in the `[DEFAULT]` sections, and the whole `[coordination]` and `[barbican]` sections. The remaining generic configuration goes into the `customServiceConfig`, or a `Secret` and is then used in `customServiceConfigSecrets`, at the `cinder: template:` level.
-. Determine if there is any scheduler-specific configuration and add it to the
-`customServiceConfig` section in `cinder: template: cinderScheduler`.
-. Determine if there is any API-specific configuration and add it to the
-`customServiceConfig` section in `cinder: template: cinderAPI`.
-. If the {block_storage} backup is deployed, add the {block_storage} backup configuration options to `customServiceConfig`, or to a `Secret` that you can add to `customServiceConfigSecrets`, at the `cinder: template:
-cinderBackup:` level. Remove the `host` configuration in the `[DEFAULT]` section to support multiple replicas in the future.
+. Determine what part of the configuration is generic for all the Block Storage services and remove anything that would change when deployed in {rhocp_long}, such as the `connection` in the `[database]` section, the `transport_url` and `log_dir` in the `[DEFAULT]` sections, the whole `[coordination]` and `[barbican]` sections. The remaining generic configuration goes into the `customServiceConfig` option, or a `Secret` custom resource (CR) and is then used in the `customServiceConfigSecrets` section, at the `cinder: template:` level.
```

Comment:
The first sentence says, “...when deployed in Red Hat OpenShift Container Platform,” but is that right? Or should it be RHOSO instead of "Red Hat OpenShift Container Platform"?